docs/Release_notes.md

```diff
@@ -225,35 +225,32 @@ In addition to the security fixes above, we have implemented AWS security featur
    - webappopen.connectedvcs.com 
    - webapptest.connectedvcs.com (test site) 
 
-Version 2.0.0, released April 25th, 2024
+Version 2.0.0, released April 30th, 2024
 ----------------------------------------
 
 ### **Summary**
 
-In this release, the MAP (also known as ISD) tool packages have been updated to use the open-source ASN1C compiler. These updates are validated with new and enhanced unit tests. UI updates include the removal of TIM and Message Validator buttons from the landing page, along with updated text in the ASN.1 text box of the MAP tool. After removing all traces of the proprietary tool, a public GitHub repository was created to release the code. All code has been added to this new repo, the README has been updated with build instructions, and the CI/CD pipeline has been revised. Lastly, a production environment has been established, and the code is now live at https://webappopen.connectedvcs.com. 
+In this release, MAP tool packages based on opensource ASN1c compiler is updated to support intersection fields that is validated with new and updated unit tests. UI updates include removal of TIM and Message Validator buttons along with an updated text in the ASN.1 text box of ISD tool. After removing all the traces to the proprietary tool, a public GitHub repo is created to release the code to public. All the code is added to this new repo, updated README with build instructions, and updated CI/CD pipeline. Lastly, production environment is created and the code is released at https://webappopen.connectedvcs.com. 
 
-This release of the Map Tool introduces several significant updates:
+This release includes the several features related to Map tool:
 
-- **<ins> Integration of Open-Source ASN1C Compiler:</ins>** After integrating the open-source ASN1c compiler, the message-builder and map-encoder packages have been updated to support all mandatory and selected optional intersection fields used in the existing MAP tool.
+- After integrating the opensource ASN1c compiler, message-builder and map-encoder code is updated to support all the mandatory intersection fields along with the optional fields used in existing MAP tool. 
+- Added and updated unit tests in message-builder and map-encoder to test the new code.
+- The message builder package is updated to record the generated MAP message and its encoded hex string in a log file that will be saved on the server for verification purposes. 
+- In the UI, removed buttons to TIM and message validator on landing page since those are not implemented in this version of the open-source tool; updated the ASN.1 text box of the ISD tool that it will not be populated in this version of the Tool.

```

Comment:
Let us add hyperlink to these two modules (**TIM** and **message validator**) to help with navigation. _Note: I do not mean the source code of the UI buttons._